presign/base/migrations/0007_participantlogevent.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.2.5 on 2023-09-30 13:32
```

Comment:
We need to coordinate this migration with https://github.com/AlpakaInfraCrew/presign/pull/60